### ISSUE TITLE
Doesn't work on Raspberry

### ISSUE THREAD

**Issue by lordofscripts** [TheCreeper/go-notify#5]
I am using a Raspberry π (Debian) and tried to run it, but I got this:

```
The name org.freedesktop.Notifications was not provided by any .service files.

```

**Comment by lordofscripts** [comment]
I installed "`dunst`" (notification daemon already installed) and the error disappeared.

However, it still doesn't work. I added the no-expiration, a sound hint (I can play sounds on my machine) but it simply executes silently. After many consecutive runs at best I see something flashing on the notification area for what seems like a millisecond because that's what I get at best.

**Comment by lordofscripts** [comment]
Okay, I got it to work but the project README doesn't sufficiently document how to use it. That's the purpose of publishing it on GitHub.

So, on my Raspberry π (Linux Debian-based):

- `notification-daemon` was installed but not sufficient to run. It would go **fatal** with a "_no freedesktop service error_."
- Installed `dunst` (_didn't try other alternatives_). Now it would run but the notification couldn't be seen.
- I suggest users to run the test and see what capabilities are reported by the unit test.

I reworked the code to get it working. Basically if you spit the notification out and immediately close it, the user won't see anything; therefore it was a bad example. At the minimum for a working example, a `Sleep` of a few seconds should be added. However, I am using channels & timeout to decouple the notification from the closing of the same should the user not click on it.

So I suggest:

- a better example (not just a unit test)
- more complete examples, like formatting the notification body.

Anyway, great thing to add in my GO programs.
